components/Home.jsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react";
+import { fetchAllArticles } from "../api";
+import ArticleCard from "./ArticleCard";
+
+function Home() {
+  const [articleList, setArticleList] = useState("");
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    fetchAllArticles().then(({ data: { articles } }) => {
+      setArticleList(articles);
+      setIsLoading(false);
+    });
+  }, []);
+
+  if (isLoading === true) return <p>Loading...</p>;
+  return (
```

Comment:
I would put this return in an else statement to stop your app trying to render anything while it's still loading